regparser/grammar/atomic.py

```diff
@@ -9,7 +9,7 @@
 
 lower_p = (
     Suppress("(")
-    + Word(string.ascii_lowercase, max=1).setResultsName("p1")
+    + Regex(r"[ivx]{1}|[a-hj-uwyz]{1,2}").setResultsName("p1")
```

Comment:
Yeah. I fear when we have (x), (y), (z), (aa), (bb), ... (hh), (ii), (jj). But so far I haven't seen it get that far.